033-OSSDatabaseMigration/Coach/00-prereqs.md

```diff
@@ -13,27 +13,28 @@ There are a lot of opportunities for things to go wrong in this step especially
 
     The location may also potentially need to be made unique if you are getting errors related to quota.
 
-*  Even though it's commented out, the AKS CLI tools may need to be installed if the attendee is not using Azure Cloud Shell
-
-* Both the Azure CLI and Helm will need to be installed. These are included with Azure Cloud Shell but will need to be downloaded separately otherwise
+* Azure CLI, AKS CLI and Helm will need to be installed. These are included with Azure Cloud Shell but will need to be downloaded separately if not using Cloud Shell.
 
 * Attendees should not be using the public IP address of the PostgreSQL/MySQL containers when configuring the application.
 
 * The attendees may make mistakes with the datasourceURL and username/password for PostgreSQL/MySQL. If the application isn't working, have the attendees check the log:
 
 ```bash
     kubectl -n {infrastructure.namespace goes here} logs deploy/contosopizza --tail=5000
+    OR
+    kubectl -n {infrastructure.namespace goes here} logs deploy/contosopizza
 ```
 
-* Google ReCaptcha will not work by default which means they will not be able to register as a user in the application. The attendee will need to generate their own. The instructions are here: https://github.com/pzinsta/pizzeria
-
-* The approximate cost for this hack is about $15/day:
-    AKS: $90/month
-    Azure DB for MySQL: $153/month
-    Azure DB for PostgreSQL: $139/month
+* [Optional] Google ReCaptcha will not work by default which means they will not be able to register as a user in the application. The attendee will need to generate their own. The instructions are here: https://github.com/pzinsta/pizzeria. The IP address for each web application will need to be registered as a domain in Google reCaptcha. 
 
-    Total: $382/month (or $13 + any incidental charges for storage, private endpoints, etc.)
+* The approximate cost for this hack is about $15/day (using Azure Pay as You Go pricing):
```

Comment:
I recommend moving this to the top of this page in the coach guide in a future update.  Put these cost requirements up front.